numpy/core/src/multiarray/calculation.c

```diff
@@ -1186,6 +1186,13 @@ PyArray_Conjugate(PyArrayObject *self, PyArrayObject *out)
     }
     else {
         PyArrayObject *ret;
+        if (!PyArray_ISNUMBER(self)) {
+            if (DEPRECATE("attempting to conjugate non-numeric dtype; this "
```

Comment:
One last thing - this needs a comment like `Numpy 1.13, 2017-05-04` by the deprecation. Sorry for not spotting that in the last review